addons/point_of_sale/models/pos_session.py

```diff
@@ -634,7 +634,7 @@ def _get_diff_account_move_ref(self, payment_method):
     def _get_diff_vals(self, payment_method_id, diff_amount):
         payment_method = self.env['pos.payment.method'].browse(payment_method_id)
         diff_compare_to_zero = self.currency_id.compare_amounts(diff_amount, 0)
-        source_account = payment_method.outstanding_account_id
+        source_account = payment_method.outstanding_account_id or self.company_id.transfer_account_id
```

Comment:
@william-andre Could you have a look here ? You changed this line here (https://github.com/odoo/odoo/commit/01b87f1230beac0568f4e3b1b76e547909506892#diff-86db45bc09231ecbc49a64405de92e0a022a6833980f46a9111501f8c8832760). But here I think we should always have a source account to track losses when counting cash in Pos sessions. Thanks !

Reply:
I changed this line because the fields was deleted.
I remember that this was one of the problematic changes, but I don't remember the decision made in the end.

I'm not sure if the transfer account is better than forcing to put an outstanding account on the payment method.

Reply:
You should be able to use `<account.payment>._get_outstanding_account` instead